File: app/src/main/java/com/site/vs/videostation/entity/DetailEntity.java
package com.site.vs.videostation.entity;

import java.io.Serializable;
import java.util.List;

/**
 * @author dxplay120
 * @date 2016/12/17
 */
public class DetailEntity implements Serializable {

    public String id;
    public int type;
    public String gold;
    public int year;
    public String pic;
    public String keywords;
    public String content;
    public String actor;
    public String director;
    public String area;
    public String name;
    public List<Origin> vod_url_list;
    public List<Near> near_list;
    public List<Comment> comment_list;

    public static class Origin implements Serializable {
        public OriginName origin;
        public List<Play> list;


        public static class Play implements Serializable {
            public String play_name;
            public String play_url;
        }

        public static class OriginName implements Serializable {
            public String name;
            public String img_url;
        }
    }

    public static class Comment implements Serializable {
        public String content;
        public String creat_at;
        public String username;
    }

    public static class Near implements Serializable {
        public String id;
        public String name;
        public String pic;
        public String title;
        public int year;
        public String area;
        public String actor;
    }
}

File: app/src/main/java/com/site/vs/videostation/entity/DetailEntity.kt
package com.site.vs.videostation.entity

import java.io.Serializable

/**
 * @author dxplay120
 * @date 2016/12/17
 */
class DetailEntity : Serializable {
    var id: String? = null
    var type: Int = 0
    var gold: String? = null
    var year: Int = 0
    var pic: String? = null
    var keywords: String? = null
    var content: String? = null
    var actor: String? = null
    var director: String? = null
    var area: String? = null
    var name: String? = null
    var vod_url_list: List<Origin>? = null
    var near_list: List<Near>? = null
    var comment_list: List<Comment>? = null

}



class Play : Serializable {
    var play_name: String? = null
    var play_url: String? = null
}

class OriginName : Serializable {
    var name: String? = null
    var img_url: String? = null
}



class Origin : Serializable {
    var origin: OriginName? = null
    var list: List<Play>? = null


}

class Comment : Serializable {
    var content: String? = null
    var creat_at: String? = null
    var username: String? = null
}

class Near : Serializable {
    var id: String? = null
    var name: String? = null
    var pic: String? = null
    var title: String? = null
    var year: Int = 0
    var area: String? = null
    var actor: String? = null
}